frappe/__init__.py

```diff
@@ -1737,7 +1725,7 @@ def setup_module_map(include_all_apps: bool = True) -> None:
 	if not local.app_modules:
 		local.app_modules = {}
 		if include_all_apps:
-			apps = get_all_apps(with_internal_apps=True)
```

Comment:
This is separate from `sites_path` removal. Don't mix two changes, or at least mention in PR desc / depracation records. 

https://github.com/frappe/frappe/wiki/Migrating-to-nightly-version

Reply:
Yeah, this is actually exacted by the sister PR as removing `site_path` broke without this change, this api was used uncleanly and potentially buggy or only worked via a very contrived coincidence of site being `"'` or something like that.